lib/components/home/artist_rails/artist_card.js

```diff
@@ -33,6 +33,7 @@ class ArtistCard extends React.Component {
     ARTemporaryAPIModule.setFollowStatus(true, this.props.artist._id, (error, following) => {
       if (error) {
         __DEV__ ? console.error(error) : console.log(error)
+        this.setState({ processingChange: false })
```

Comment:
Would it make sense to set `processingChange: true` before the networking request? I know these are completion blocks and `setState` will happen right away but conceptually doesn't the change-processing start as soon as the button is pressed?